nextflow-modules/modules/cron/main.nf

```diff
@@ -1,28 +1,28 @@
 process copy_to_cron {
-  tag "${sampleName}"
+  tag "${sampleID}"
   scratch params.scratch
 
   input:
-    tuple val(sampleName), path(yaml), path(cdm)
+    tuple val(sampleID), path(yaml), path(cdm)
 
   output:
-    tuple val(sampleName), path(yamlOutput), emit: yaml
-    tuple val(sampleName), path(cdmOutput) , emit: cdm
+    tuple val(sampleID), path(yamlOutput), emit: yaml
+    tuple val(sampleID), path(cdmOutput) , emit: cdm
 
   when:
     params.cronCopy
 
   script:
-    yamlOutput = "${sampleName}_bonsai.yaml"
-    cdmOutput = "${sampleName}.cdm"
+    yamlOutput = "${sampleID}_bonsai.yaml"
+    cdmOutput = "${sampleID}.cdmpy"
     """
     cp ${yaml} ${yamlOutput}
     cp ${cdm} ${cdmOutput}
     """
 
   stub:
-    yamlOutput = "${sampleName}_bonsai.yaml"
-    cdmOutput = "${sampleName}.cdm"
+    yamlOutput = "${sampleID}_bonsai.yaml"
+    cdmOutput = "${sampleID}.cdmpy"
```

Comment:
Ditto: Is the `py` ending here intended?

Reply:
yep